src/main/java/uk/gov/companieshouse/model/validator/EndDateValidator.java

```diff
@@ -26,9 +26,13 @@ public String validateEndDate(Appeal appeal) {
         if (!continued && rawEndDate == null) {
             return EMPTY_END_DATE;
         }
-        if (!rawEndDate.isEmpty() && continued) {
+
+        if (rawEndDate != null  && continued) {
```

Comment:
If you want to be on the safe side and check for an empty string as well you could do _if ((rawEndDate != null && !rawEndDate.isEmpty()) && continued)_ This will check rawEndDate is not null and whether it's not empty